mvnw

```diff
@@ -21,288 +21,219 @@
 # ----------------------------------------------------------------------------
 # Apache Maven Wrapper startup batch script, version 3.2.0
 #
-# Required ENV vars:
```

Comment:
The wrapper should be updated in a separate commit, or even in a separate PR, not to block this one. There was a prior discussion about removing the wrapper: https://github.com/trinodb/trino/pull/19186

What would be the argument for switching to a script-only wrapper? One argument raised in the PR I linked above is to avoid downloading it every time in the CI.

Reply:
the "normal" wrapper disobey some env variables and/or settings (unsure precisely), while "only-script" simply delegates directly to mvn script of distro, and supports all that maven distro should support. Some of the reported issues https://issues.apache.org/jira/browse/MWRAPPER-110